digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -291,7 +291,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     * @param linkIds the linkIds to process
     * @param roadNumber the roadNumber to apply/was applied to said linkIds
     * @param roadPartNumber the roadPartNumber to apply/was applied to said linkIds
-    * @param newLinks the whole newProjectLinks
+    * @param existingLinks the whole newProjectLinks
```

Comment:
existingLinks the whole *existing* ProjectLinks :p

Reply:
Are you implying that newProjectLinks DO NOT EXIST GOOD SIR?!